SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
```

Comment:
This statement sounds like it's an instruction for creating the security policy. Consider changing to "Versions currently being supported with security updates are listed below."